federation/pkg/dnsprovider/providers/google/clouddns/internal/stubs/rrsets_list_call.go

```diff
@@ -19,19 +19,30 @@ package stubs
 import (
 	"google.golang.org/api/googleapi"
 	"k8s.io/kubernetes/federation/pkg/dnsprovider/providers/google/clouddns/internal/interfaces"
+	"strings"
 )
 
 // Compile time check for interface adherence
 var _ interfaces.ResourceRecordSetsListCall = &ResourceRecordSetsListCall{}
 
 type ResourceRecordSetsListCall struct {
-	Response_ *ResourceRecordSetsListResponse
-	Err_      error
-	Name_     string
-	Type_     string
+	Response_   *ResourceRecordSetsListResponse
+	Err_        error
+	Name_       string
+	Type_       string
+	MaxResults_ int64
 }
 
 func (call *ResourceRecordSetsListCall) Do(opts ...googleapi.CallOption) (interfaces.ResourceRecordSetsListResponse, error) {
+	if len(call.Name_) > 0 {
```

Comment:
It's not at all clear why all this additional logic is required here.  Please add a comment to explain, or remove it.